test-fixture.html

```diff
@@ -313,9 +315,20 @@
     });
 
   try {
-    document.registerElement('test-fixture', {
-      prototype: TestFixturePrototype
-    });
+    if (window.customElements) {
+      function TestFixture() {
+        if (window.Reflect) {
+          return Reflect.construct(HTMLElement, [], TestFixture)
+        }
+        return HTMLElement.call(this) || this;
+      }
+      TestFixture.prototype = TestFixturePrototype;
+      customElements.define('test-fixture', TestFixture);
```

Comment:
There are versions of Chrome w/ Reflect.construct, not not CustomElements v1. I think how you call super, and how you register the element should probably be separate concerns.